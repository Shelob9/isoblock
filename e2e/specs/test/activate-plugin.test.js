/**
 * WordPress dependencies
 */
import {
  insertBlock,
  getEditedPostContent,
  createNewPost,
  activatePlugin
} from "@wordpress/e2e-test-utils";

describe("Button", () => {
  beforeEach(async () => {
    await activatePlugin("isoblock/isoblock.php");
  });

  it("can jump focus back & forth", async () => {
    await createNewPost();
    await insertBlock("Hydrate Demo");

    expect(await getEditedPostContent()).toMatchSnapshot();
  });
});
