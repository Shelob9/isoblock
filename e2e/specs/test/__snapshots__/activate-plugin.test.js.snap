// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button can jump focus back & forth 1`] = `
"<!-- wp:josh/isoblock {\\"blockId\\":\\"5a2065d8-9d9a-434e-80f8-9c0823dc92c0\\"} -->
<form class=\\"wp-block-josh-isoblock\\"><div><label for=\\"5a2065d8-9d9a-434e-80f8-9c0823dc92c0\\">Form Field</label><input value=\\"Default Default Value\\" id=\\"5a2065d8-9d9a-434e-80f8-9c0823dc92c0\\" class=\\"isoblock-field\\"/></div></form>
<!-- /wp:josh/isoblock -->"
`;
